tsconfig.json

```diff
@@ -14,38 +14,14 @@
     "target": "es2015",
     "module": "es2020",
     "lib": ["es2018", "dom"],
+    "skipLibCheck": true,
```

Comment:
What does this one do?

Reply:
Skips checking of TypeScript for libraries inside `node_modules`. Had to add it after installing libs like `lto-transactions` that require a different TS version, so it gives out warnings. Ideally the lib would update to same TS version, but as long as lib works, I don't see an issue